llm/podscript/get_content.py

```diff
@@ -0,0 +1,40 @@
+import os
+import openai
+from dotenv import load_dotenv, find_dotenv
+from langchain_community.document_loaders.blob_loaders.youtube_audio import (
+    YoutubeAudioLoader,
+)
+from langchain_community.document_loaders.generic import GenericLoader
+from langchain_community.document_loaders.parsers.audio import (
+    OpenAIWhisperParser,
+    OpenAIWhisperParserLocal,
+)
+from langchain_community.document_loaders import UnstructuredXMLLoader
+from langchain.text_splitter import RecursiveCharacterTextSplitter, SentenceTransformersTokenTextSplitter
+
+
+_ = load_dotenv(find_dotenv())
+
+openai.api_key  = os.environ['OPENAI_API_KEY']
+
+
+# transcript data
+local = False # TODO: failed in using local parsing
```

Comment:
what's the value for? if not related in this PR, just remove it

Reply:
As mentioned in the [Langchain API documentation](https://python.langchain.com/docs/integrations/document_loaders/youtube_audio), there are two types of parsing methods: one for public cloud and another for a private cloud for privacy concerns. I'd like to keep both for better scalability if we decide to use a private cloud in the future.